plugins/Actions/DataTable/Filter/Actions.php

```diff
@@ -35,6 +35,7 @@ public function __construct($table, $isPageTitleType)
     public function filter($table)
     {
         $table->filter(function (DataTable $dataTable) {
+            $urlPrefix = $dataTable->getMetadata('site')->getMainUrl();
```

Comment:
When viewing an existing report **flattened** this throws a `Call to a member function getMainUrl() on boolean`